web/src/lib/components/pages/record/record.tsx

```diff
@@ -451,9 +451,9 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
         <span id="why-profile-title">Why a profile?</span>
         <div id="why-profile">
           <p id="why-profile-text">
-            Copy explaining value of profile &amp; demographic capture: Lorem
-            ipsum dolor sit amet, consectetur adipiscing elit. Nulla id orci
-            dui.
+            By providing some information about yourself, the audio data you
+            submit to Common Voice will be more useful to Speech Recognition
+            engines that use this data to improve their accuracy.
```

Comment:
While this is all true and a LGTM is fine, I don't think the "spirit" is captured.

The demographic information provided will help speech recognition engines handle previously overlooked accents, ages, and genders. Effectively allows future engines to better address the full spectrum of spoken language.

Reply:
Yeah, I agree we haven't captured the spirit of Common Voice around its inclusivity. However, I think it's more important for that message to show up front and center on the homepage, which we are currently working on.

Reply:
OK, then LGTM